resources/views/donation_form.blade.php

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
```

Comment:
use blade layout instead of copy pasting it directly

https://laravel.com/docs/8.x/blade#building-layouts